tests/Jwt/JwtV2ValidatorTest.php

```diff
@@ -98,21 +98,32 @@ public function it_verifies_the_basic_claims_via_the_decoratee(): void
      */
     public function it_verifies_the_permission_to_use_entry_api_if_azp_claim_is_present(): void
     {
-        $tokenWithPermission = $this->createTokenWithClaims(
+        $accessTokenWithPermission = $this->createTokenWithClaims(
```

Comment:
Should be `$userAccessTokenWithPermissions` to avoid confusion since it's not clear if this is an instance of a user access token or a client access token like this